### DISCUSSION TITLE
Inline code text/plot output (with R)

### DISCUSSION THREAD

**Discussion by stefanocoretta** [quarto-dev/quarto-nvim#123] [Q&A] [answered]
Hi! I am desperately looking to find a way of getting inline code output (text/plot) as you would in RStudio.

Is that possible at all? (I am thinking R and .qmd files specifically)

**Comment by jmbuhr** [comment, marked as answer]
Yes, with https://github.com/benlubas/molten-nvim

**Reply by stefanocoretta** [reply]
Oh wow! Thanks! I should also watch all of your videos 😅

**Reply by jmbuhr** [reply]
ah, yes, there is a video that mentions it, but that was before I started giving them helpful (and more catchy) titles.
